### PR TITLE
Add an ALTCHA noscript warning message

### LINKED ISSUE #7680: ALTCHA Antispam: Add < noscript > message
### Affected version(s)

5.4

### Description

[From the Altcha Docs](https://altcha.org/docs/website-integration/): "When integrating the widget into website forms, consider that users without JavaScript enabled in their browsers will be unable to utilize or submit forms protected by the widget. Ensure your audience is aware of this requirement for seamless form submission."

Current situation in Contao: If JavaScript is disabled, the Altcha form element is not shown. There is no warning message that JavaScript is required to process the form.

After the user tries to submit the form, an error message is displayed: "ALTCHA Antispam-Überprüfung fehlgeschlagen. Sind Sie ein Mensch oder ein Roboter?" This error message doesn't give a hint that the user have to enable Javascript in the browser. 

I think it would be useful to add a < noscript > message to the default form_altcha.html5 template to inform users that “Javascript must be enabled for this form to work” or something similar.

Furthermore, I think the word “ALTCHA” should be omitted from the error messages. This term is not relevant or helpful for the visitor who wants to send the form.

### PR DESCRIPTION
**Pull request by CMSworker**
Fixes #7680

### CODE REVIEW

File: core-bundle/contao/languages/en/default.xlf [hunk 1]
```diff
@@ -281,8 +281,11 @@
       <trans-unit id="ERR.noAdminEmail">
         <source>There is no administrator e-mail address set, please contact the site administrator.</source>
       </trans-unit>
+      <trans-unit id="ERR.altchaJavascriptRequired">
```
Comment (xchs) [inline core-bundle/contao/languages/en/default.xlf]:
```suggestion
      <trans-unit id="ERR.altchaJavaScriptRequired">
```

File: core-bundle/contao/languages/en/default.xlf [hunk 2]
```diff
@@ -281,8 +281,11 @@
...
+        <source>Javascript is currently disabled or blocked. Without Javascript, the anti-spam verification will fail. Please enable Javascript temporarily for form submission.</source>
```
Comment (xchs) [inline core-bundle/contao/languages/en/default.xlf]:
```suggestion
        <source>JavaScript is currently disabled or blocked. Without JavaScript, the anti-spam verification will fail. Please enable JavaScript temporarily for form submission.</source>
```

File: core-bundle/contao/templates/forms/form_altcha.html5 [hunk 3]
```diff
@@ -16,13 +16,14 @@ $GLOBALS['TL_BODY'][] = sprintf('<script src="%s" type="module"></script>', $thi
 <?php $this->endblock(); ?>
 
 <?php $this->block('field'); ?>
-    <?php if ($this->hasErrors()): ?>
-      <p class="error"><?= $this->getErrorAsString() ?></p>
-    <?php endif; ?>
+  <noscript><p class="warning"><?= $this->trans('ERR.altchaJavascriptRequired') ?></p></noscript>
```
Comment (xchs) [inline core-bundle/contao/templates/forms/form_altcha.html5]:
```suggestion
  <noscript><p class="warning"><?= $this->trans('ERR.altchaJavaScriptRequired') ?></p></noscript>
```

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @CMSworker.

**Pull request merged**